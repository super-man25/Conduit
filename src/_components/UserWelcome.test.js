import React from 'react';
import { UserWelcome } from './';
import renderer from 'react-test-renderer';

const testUser = { id: 1, firstName: 'John', lastName: 'Smith' };

it('renders correctly with no props', () => {
  const tree = renderer
    .create(<UserWelcome></UserWelcome>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly with a valid user prop', () => {
  const tree = renderer
    .create(<UserWelcome user={testUser}></UserWelcome>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
