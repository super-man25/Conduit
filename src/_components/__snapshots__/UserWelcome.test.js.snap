// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with a valid user prop 1`] = `
<div
  className="sc-caSCKo gXaPkV"
/>
`;

exports[`renders correctly with no props 1`] = `
<div
  className="sc-caSCKo gfIZkt"
/>
`;
